pkg/lib/ratelimit/bucket.go

```diff
@@ -0,0 +1,9 @@
+package ratelimit
```

Comment:
Should this package live in `pkg/util` instead?

Reply:
The package depends on Redis, which is in `pkg/lib`.